src/smart-components/role/add-role-new/add-permissions-template.js

```diff
@@ -1,14 +1,17 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Chip, ChipGroup, Text, TextContent, Title } from '@patternfly/react-core';
+import { Chip, ChipGroup, Text, TextContent, Title, Button, Popover } from '@patternfly/react-core';
 import useFormApi from '@data-driven-forms/react-form-renderer/dist/cjs/use-form-api';
-
+import QuestionCircleIcon from '@patternfly/react-icons/dist/js/icons/outlined-question-circle-icon';
 import './add-role-wizard.scss';
 
 const AddPermissionTemplate = ({ formFields }) => {
   const formOptions = useFormApi();
   const [selectedPermissions, setSelectedPermissions] = useState(formOptions.getState().values['add-permissions-table'] || []);
 
+  const unresolvedSplats =
+    formOptions.getState().values?.['copy-base-role']?.applications?.filter((app) => selectedPermissions?.find(({ uuid }) => uuid.includes(app))) ||
```

Comment:
I think that logic here should be negated. You are selecting resolved permissions. When i tried it patch has 1 resolved 1 unresolved so it seems to work. But for approval, that has all permissions resolved it also shows message.

Reply:
Aaaah, yeah I've been messing around with the condition and pushed the wrong one. Good catch!